src/collective/revisionmanager/Extensions/Install.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from collective.revisionmanager.config import PROJECTNAME
+from plone import api
+
+
+# see: http://docs.plone.org/manage/troubleshooting/manual-remove-utility.html
+def _remove_persistent_utility(portal):
+    from collective.revisionmanager.interfaces import IHistoryStatsCache
+    sm = portal.getSiteManager()
+    util = sm.getUtility(IHistoryStatsCache)
+    sm.unregisterUtility(provided=IHistoryStatsCache)
+    del util
+    sm.utilities.unsubscribe((), IHistoryStatsCache)
+    del sm.utilities.__dict__['_provided'][IHistoryStatsCache]
+    del sm.utilities._subscribers[0][IHistoryStatsCache]
+    # logger.info('IHistoryStatsCache persistent utility successfully removed')
+
+
+def uninstall(portal, reinstall=False):
```

Comment:
I tested manually and it was not working without the addition of this module but I can double check.